packages/dbml-core/src/parse/ANTLR/ASTGeneration/postgres/PostgreSQLLexerBase.js

```diff
@@ -0,0 +1,20 @@
+import antlr4 from 'antlr4';
+
+// original (C#): https://github.com/antlr/grammars-v4/blob/master/sql/postgresql/CSharp/PostgreSQLLexerBase.cs
+// Base class required by the generated lexer
+export default class PostgreSQLLexerBase extends antlr4.Lexer {
+}
+
+// helper functions required by the generated grammar
+// To avoid error, we declare their signature and import them in the generated PostgreSQLLexer.js
```

Comment:
You should mention that these methods were defaulted in C# but they were not defaulted in Javascript. Do these methods affect the result of the parser?

Reply:
Updated the comment in https://github.com/holistics/dbml/pull/416/commits/5acb21aed42fee4aa1c5c3cc69a6d60a431920ab.

> Do these methods affect the result of the parser?

According to my testing, no.